pages/common/pngquant.md

```diff
@@ -0,0 +1,36 @@
+# pngquant
+
+> PNG converter and lossy image compressor.
+> More information: <https://pngquant.org/>.
+
+- Compress a PNG as much as possible and write result to a new file:
+
+`pngquant {{path/to/file.png}}`
+
+- Compress a PNG and override original:
+
+`pngquant --ext .png -f {{path/to/file.png}}`
+
+- Try to compress a PNG with custom quality (skip if below the min value):
+
+`pngquant -Q {{0-100}} {{path/to/file.png}}`
+
+- Compress a PNG with the number of colors reduced to 64:
```

Comment:
There are multiple fixes available: please chose the most appropriate one.

```suggestion
- Compress a specific PNG with the number of colors reduced to 64:
```

Optional change: `a specific` may sound better.

```suggestion
- Compress a specific .png with the number of colors reduced to 64:
```

Optional change: you can use `.ext` extension format instead of `EXT` one.